wiki/Skinning/Compendium/en.md

```diff
@@ -129,88 +156,123 @@ Compendium of all completed osu! skins.
+- [Neon Genesis Evangelion](https://osu.ppy.sh/community/forums/topics/417739) by [Elzapatoverde](https://osu.ppy.sh/users/3717733)![][o!s]![][o!c]![][o!m]
 - [NeOsu](https://osu.ppy.sh/community/forums/topics/83087) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]
-- [NeOsu Symphonia](https://osu.ppy.sh/community/forums/topics/183350) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!m]
-- [NeOsu 5 (NeOsu Surge)](https://osu.ppy.sh/community/forums/topics/202838) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
-- [NeOsu 3](https://osu.ppy.sh/community/forums/topics/132898) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
 - [Neosu Reloaded!! Fan Remix](https://osu.ppy.sh/community/forums/topics/658953) by [My Angel Digi](https://osu.ppy.sh/users/4543414)![][o!s]![][o!m]
-- [NeOsu 2](https://osu.ppy.sh/community/forums/topics/110915) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
 - [Neosu Reloaded!! Fan Remix](https://osu.ppy.sh/community/forums/topics/658953) by [My Angel Digi](https://osu.ppy.sh/users/4543414)![][o!s]![][o!m]
+- [NeOsu Symphonia](https://osu.ppy.sh/community/forums/topics/183350) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!m]
+- [NeOsu TIARA](https://osu.ppy.sh/community/forums/topics/399690) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
+- [NeOsu 2](https://osu.ppy.sh/community/forums/topics/110915) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
+- [NeOsu 3](https://osu.ppy.sh/community/forums/topics/132898) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
+- [NeOsu 5 (NeOsu Surge)](https://osu.ppy.sh/community/forums/topics/202838) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
 - [Neo Street](https://osu.ppy.sh/community/forums/topics/332576) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!m]
+- [Nero](https://osu.ppy.sh/community/forums/topics/651827) by [dlwnstns99](https://osu.ppy.sh/users/4585661)![][o!s]
 - [Nexus Slate & Ivory](https://osu.ppy.sh/community/forums/topics/55948) by [Xiao](https://osu.ppy.sh/users/17894)![][o!s]
+- [Nexus](https://osu.ppy.sh/community/forums/topics/432071) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!t]![][o!c]
 - [Nikaido Yuzu](https://osu.ppy.sh/community/forums/topics/604971) by [Lathler](https://osu.ppy.sh/users/9039378)![][o!s]
+- [NOEL](https://osu.ppy.sh/community/forums/topics/651827) by [dlwnstns99](https://osu.ppy.sh/users/4585661)![][o!s]
 - [Northen Ocean Hime](https://osu.ppy.sh/community/forums/topics/319797) by [-[Jess]-](https://osu.ppy.sh/users/4085133)![][o!s]![][o!c]
 - [Nui Harime](https://osu.ppy.sh/community/forums/topics/417954) by [carnationturtle](https://osu.ppy.sh/users/4461382)
```

Comment:
Missing mode icons.

Reply:
Nope/missing on purpose. That skin simply doesn't have any modes skinned.

Reply:
Really? There are no gameplay screenshots maybe that's why. I actually downloaded it to check. It skins osu!standard.

Says it on skin forum:
> Currently, this skin supports only osu!

Reply:
You're right. It's skinned, but ther are no screenshots. Fixed that.